test/prop_mimemail.erl

```diff
@@ -99,6 +100,80 @@ match({TypeA, SubTypeA, HeadersA, _ParamsA, BodyA},
 			  end, Bodies)
 	end.
 
+prop_quoted_printable(doc) ->
+	"Make sure quoted-printable encoder works as expected: "
+		"* No lines longer than 76 chars "
+		"* decode(encode(data)) returns the same result as original input".
+
+prop_quoted_printable() ->
+	Trim = fun(B) -> binstr:reverse(trim(binstr:reverse(trim(B)))) end,
+	?FORALL(
+	   Body,
+	   proper_types:oneof([?SIZED(Size, printable_ascii(Size * 50)),
+						   ?SIZED(Size, printable_ascii_and_cariage(Size * 50)),
+						   printable_ascii(),
+						   printable_ascii_and_cariage(),
+						   nonull_utf8(),
+						   proper_types:binary()]),
+	   begin
+		   [QPEncoded] = mimemail:encode_quoted_printable(Body),
+		   ?assertEqual(Trim(Body), Trim(mimemail:decode_quoted_printable(QPEncoded))),
```

Comment:
we still can loose heading and trailing whitespaces and tabs. Not sure if it's a bug, but doesn't sound right. Can be fixed in the following iterations.